### PR TITLE
Implementing delete

### PR DESCRIPTION
**Pull request by trisberg**

### CODE REVIEW

File: cmd/delete.go [hunk 1]
```diff
@@ -20,33 +20,85 @@ import (
...
+			var err error
+			deleteOptions.name, err = functions.FunctionNameFromPath(deleteOptions.path)
+			if (err != nil) {
+				ioutils.Errorf("Error: %v\n", err)
+				return
+			}
+		}
+
+		abs,err := functions.AbsPath(deleteOptions.path)
+		if (err != nil) {
```
Comment (dturanski) [inline cmd/delete.go]: don't need ()

File: cmd/delete.go [hunk 2]
```diff
@@ -20,33 +20,85 @@ import (
...
+			ioutils.Errorf("Error: %v\n", err)
+			return
+		}
+
+		var cmdArgs []string
+
+		if (deleteOptions.all) {
```
Comment (dturanski) [inline cmd/delete.go]: Don't need()

**Review by dturanski — COMMENTED** [review]
Other commands, e.g. `create` , `apply` interpret a single arg as the path if `-f` not provided.  This should be consistent. Other than that LGTM.

File: cmd/delete.go (hunk 2 above)
Comment (trisberg) [reply cmd/delete.go]: old habits ... ;)

**Pull request merged**